plugins/sentry-backend/src/service/sentry-api.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import express from 'express';
+import axios from 'axios';
+
+export class SentryApiForwarder {
+  constructor(private token: string) {}
+  public fowardRequest(request: express.Request, response: express.Response) {
+    const sentryUrl = request.path;
+    axios
+      .get(`https://sentry.io/${sentryUrl}`, {
+        headers: {
+          Authorization: `Bearer ${this.token}`,
```

Comment:
It's a bit risky to proxy through straight to the api without any CSRF protection. What's the scope of the token used here, and did you think about protecting against CSRF?

Reply:
My initial idea was that it's up to the backstage user to provide an appropriately scoped token (it should be read-only, solving CSRF issue). Maybe we could limit requests to `GET` as well? WDYT?